.github/workflows/ci-matrix.yml

```diff
@@ -6,6 +6,9 @@ on:
     branches:
       - master
 
+env:
```

Comment:
Pretty sure this isn't needed. It doesn't hurt anything, but according to GitHub, `GITHUB_ACTIONS` is always set by the infrastructure in an Actions builder.